README.md

```diff
@@ -9,6 +9,10 @@ Included are several well-known Traits and Specifications for use in
 OpenAssetIO hosts and managers. For more information on this mechanism,
 see the [OpenAssetIO docs](https://thefoundryvisionmongers.github.io/OpenAssetIO/).
 
+MediaCreation is a fully generated component,
```

Comment:
[fix] Switch to Python terminology
```suggestion
MediaCreation is an automatically generated Python package,
```

Reply:
Man i never get this sort of stuff right.